ProjectSidewalk/SidewalkWebpage
# Resolves #1460 Track "Jump" dialog radio option

Resolves #1460 

- When a user clicks on one of the options in the "Jump" dialog, the name of the option is included in the ClickRadio event

![](https://user-images.githubusercontent.com/7133966/52766328-b60a9180-2fdb-11e9-8ba2-a9fc82c494a3.png)
![](https://user-images.githubusercontent.com/7133966/52766334-bd319f80-2fdb-11e9-8d35-b229abf969de.png)

To test:
- Check that when you click on one of the options in the "Jump" dialog, that the option you clicked is recorded in the `audit_task_interaction` table
